### PR TITLE
code smell

### PR DESCRIPTION
**Pull request by Vidyaaa24**
Build and Run working fine

### CODE REVIEW

File: src/main/java/com/iemr/tm/service/common/transaction/CommonDoctorServiceImpl.java [hunk 1]
```diff
@@ -83,9 +84,7 @@
 import com.iemr.tm.utils.mapper.OutputMapper;
 
 /***
- * 
  * @author NE298657
```
Comment (Anjana2310) [inline src/main/java/com/iemr/tm/service/common/transaction/CommonDoctorServiceImpl.java:87]: Author to be removed

File: src/main/java/com/iemr/tm/service/dataSyncActivity/DataSyncRepository.java [hunk 2]
```diff
@@ -36,9 +36,7 @@
 import com.iemr.tm.repo.syncActivity_syncLayer.SyncUtilityClassRepo;
 
 /***
- * 
  * @author NE298657
```
Comment (Anjana2310) [inline src/main/java/com/iemr/tm/service/dataSyncActivity/DataSyncRepository.java:39]: Author to be removed

File: src/main/java/com/iemr/tm/service/foetalmonitor/FoetalMonitorServiceImpl.java [hunk 3]
```diff
@@ -130,9 +125,6 @@ public int updateFoetalMonitorData(FoetalMonitor foetalMonitorDataOutside) throw
 			foetalMonitorDataOutside.setResultState(true);
 			foetalMonitorDataOutside.setDeleted(foetalMonitorFetchDataDB.getDeleted());
 
-			// need to write the code for changing the report path data to base 64 and save
-			// it in DB
-
 			String filePath = generatePDF(foetalMonitorDataOutside.getReportPath());
```
Comment (Anjana2310) [inline src/main/java/com/iemr/tm/service/foetalmonitor/FoetalMonitorServiceImpl.java:128]: Add the comment back

File: src/main/java/com/iemr/tm/service/generalOPD/GeneralOPDServiceImpl.java [hunk 4]
```diff
@@ -76,9 +74,7 @@
 import com.iemr.tm.utils.mapper.InputMapper;
 
 /***
- * 
  * @author NE298657
```
Comment (Anjana2310) [inline src/main/java/com/iemr/tm/service/generalOPD/GeneralOPDServiceImpl.java:77]: Author to be removed

**Review by Anjana2310 — COMMENTED** [review]
@Vidyaaa24 Please address the comments

**Review by swetatech24 — APPROVED** [review]
LGTM

**Pull request merged**